sdk/keyvault/keyvault-certificates/src/certificatesModels.ts

```diff
@@ -787,11 +785,6 @@ export interface CertificateIssuer {
  * An interface representing optional parameters for CertificateClient paged operations passed to {@link listPropertiesOfCertificates}.
  */
 export interface ListPropertiesOfCertificatesOptions extends coreHttp.OperationOptions {
-  /**
-   * Maximum number of results to return in a page. If not specified the service will return up to
-   * 25 results.
-   */
-  maxresults?: number;
```

Comment:
Where does it come from, now?

Reply:
The plan is to not use this at all. Keys and Secrets don't seem to have them either. I'll sync up with you later today 🌞